storage/backend/backends.go

```diff
@@ -26,10 +33,7 @@ var SnapshotDrivers = map[string]func() (storage.SnapshotDriver, error){
 // NewMountDriver instantiates and return a mount driver instance of the
 // specified type
 func NewMountDriver(backend, mountpath string) (storage.MountDriver, error) {
-	f, ok := MountDrivers[backend]
-	if !ok {
-		return nil, errored.Errorf("invalid mount driver backend: %q", backend)
-	}
+	f, _ := MountDrivers[backend]
```

Comment:
Cool 👍  Validation always ensures that 'backend' will be one of the allowed drivers.

https://github.com/yuva29/volplugin/blob/674460acf3628f7eb45ffdd6fbdc7e260fdc141a/config/schema.go#L34-L36

String empty checks are handled in NewDriver.

But now, I think it is better to check the existence of them as its an exported function.